### DISCUSSION TITLE
Is it possible to have a viewer application show the screen in a window?

### DISCUSSION THREAD

**Discussion by miaolinopenvox** [VirtualDrivers/Virtual-Display-Driver#90] [Ideas]
Or your suggestion is use OBS as a viewer?

**Comment by quibud** [comment]
NDI Studio Monitor

**Comment by Skyyblaze** [comment]
I thought about a similar thing but refrained from opening up an Issue thinking it wouldn't be feasible anyway. It would be great if we could basically have a game run on a virtual screen minimized for Remote Streaming while the rest of the PC can still be freely interacted with.
